src/main/webapp/script.js

```diff
@@ -0,0 +1,30 @@
+function getOriginalRecipe() {
+  const key = document.getElementById("key").value;
+  if (key !== "") {
+    fetch("/new-recipe?key=" + key).then(response => response.json()).then((recipe) => {
```

Comment:
How and when did the new recipe servlet get added? I see that you had a previous PR that included it, but you closed it before merging.

Reply:
That happened about an hour ago when I tried to solve merge conflicts and accidentally merged the previous PR somehow. I closed the PR to try and delete its branch so I could open a new PR on a copied branch, but I'm not allowed to delete it. I just forgot to reopen it, and I'll do that now.

Reply:
It's saying I have a bunch of conflicts that I have to solve on the command line, but when I copied the commands into my terminal the branch ended up being merged somehow. I'm not really sure what to do.

Reply:
Update: I think I fixed it by creating a new PR called New Recipe Servlet